src/NamingStrategy/UnderscoreNamingStrategy/CamelCaseToUnderscoreFilter.php

```diff
@@ -16,13 +17,26 @@ final class CamelCaseToUnderscoreFilter
 {
     use StringSupportTrait;
 
+    /** @var string[] $transformedFilters */
+    private $transformedFilters = [];
```

Comment:
I know you act in good faith here, but a long running process is just going to die, when/if using this component this way 😟

Reply:
This was the whole point of the PR, to cache the filters, like it is already done with the ReflectionProperties

Reply:
`ReflectionProperty[]` are a finite amount of values, because you can only have that many in memory.

the transformed strings are potentially billions/more.

Reply:
Ah I see, then you guys need to discuss the pr. It is deliberately as non static property. I thought only a static property would cause these issues.

Reply:
One thing I just noticed, the amount of ReflectionProperties and transformed strings will always be the same cause the filter gets called based on the ReflectionProperty.